packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -113,3 +116,163 @@ class DefaultTaggedText extends StatelessWidget {
     );
   }
 }
+
+/// A widget that takes two children and lays them out along one axis.
+///
+/// The user can customize the amount of space allocated to each child by
+/// dragging a divider between them.
+class Split extends StatefulWidget {
+  /// Builds a [Split] with [Axis.horizontal] direction.
+  ///
+  /// [firstChild] will be placed before [secondChild] in a [Row].
+  const Split.horizontal({
+    Key key,
+    @required Widget firstChild,
+    @required Widget secondChild,
+    double initialFirstFraction,
+  }) : this._(
+          key,
+          Axis.horizontal,
+          firstChild,
+          secondChild,
+          initialFirstFraction,
+        );
+
+  /// Builds a [Split] with [Axis.vertical] direction.
+  ///
+  /// [firstChild] will be placed before [secondChild] in a [Column].
+  const Split.vertical({
+    Key key,
+    @required Widget firstChild,
+    @required Widget secondChild,
+    double initialFirstFraction,
+  }) : this._(
+          key,
+          Axis.vertical,
+          firstChild,
+          secondChild,
+          initialFirstFraction,
+        );
+
+  const Split._(
+    Key key,
+    this.axis,
+    this.firstChild,
+    this.secondChild,
+    double initialFirstFraction,
+  )   : initialFirstFraction = initialFirstFraction ?? 0.5,
```

Comment:
Does ??= work for this case?

Reply:
No.  This is equivalent to:

```dart
this.initialFirstFraction = initialFirstFraction ?? 0.5
```

The two variables are different, even though they have the same name.